### ISSUE TITLE
Keyboard navigation with tab does not work on items with plural forms

### ISSUE THREAD

**Issue by CyrilleB79** [vslavik/poedit#796]
Hello

In a .po file, I have to translate a string plural form.
When I have selected this string, I can tab until the translation field (singular). But once I am done with this field, I cannot tab forward or shift+tab back to the other focusable items of the UI.
This issue does not occur when translating a single string with no plural form.

Could you please have a look? Are you able to confirm this issue? (else, I'll provide more detailed steps)
Thanks.

**Comment by vslavik** [comment]
> (else, I'll provide more detailed steps)

Please do (as GitHub asked you to when you created this issue). You don't even spell out basics like Poedit version or your OS...

**Comment by florianionascu7** [comment]
Hello! I'm here to confirm what Cyrille said. Please fix this accessibility bug.

**Comment by vslavik** [comment]
> Hello! I'm here to confirm what Cyrille said.

@florianionascu7  please don't post useless "me too" comments. You're just wasting everyone's time with that. You can thumbs-up the original instead.

> Please fix this accessibility bug.

Instead of directing me what to do (do you think I wouldn't know it should be fixed without your comment?), why don't you contribute patches? That would help a lot, unlike nagging...

**Comment by CyrilleB79** [comment]
@vslavik, sorry for disturbing. But is there a chance to have this issue fixed in a near future? Do you plan to work on this yourself or do you rather expect a volunteer contributor to work on it?

Or are you still expecting inputs from the reporter (i.e. myself)? If yes, let me know what please.

**Comment by CyrilleB79** [comment]
@vslavik, this issue seems to be fixed on my end with poedit 3.5.2, with the new tab control for plural forms.

You can close this issue if you agree that it is fixed.
